docs/cookbook/backend-api/creating-custom-api-endpoint.md

```diff
@@ -0,0 +1,73 @@
+# Creating Custom API Endpoint
+
+This cookbook provides you all the necessary steps that you will need for extending your [backend API](/docs/backend-api/introduction-to-backend-api.md) with a custom endpoint.
+
+To keep the cookbook as simple as possible, we will just add `/hello/<name>` endpoint that will greet the API client.
```

Comment:
I do not like the idea of this dummy solution for the cookbook. The cookbook should solve some real need. I strongly suggest exporting something real - even if this would be a list of all product descriptions with the idea to export for a translator.

Reply:
I have to disagree because as simple as the example is, it fulfills its purpose. You need to add your own endpoint and you can return anything you want and this few little steps is answer to you how to do that - four programmers agreed on this :-D

Reply:
Please discuss it a bit on Monday. I do not want to force it, but for me it is change in our current approach to cookbooks.

Reply:
Ok we will

Reply:
I support Tom in this decision - it serves programmer's needed. I need to add a new endpoint and I have no idea how to do it. There is a cookbook, I'll follow it. Ok, now I have a working endpoint that return static data. Ok, now I can use a facade to provide desired data - but this is my standard work - to program :-)

This cookbook covers only the unknown "how to add an endpoint", and that is the valuable part for developer.

I think even five programmers agreed on this - me, Tom, Rosťa (written the Doc), David (CR), Martin (tested the cookbook).

:trollface: : if you need more discussion, you have to convene whole team meeting (as only Petr and Boris didn't see it yet).

Reply:
ok, I am not happy with it, but I can live with it :-)